packages/api/lib/http.service.ts

```diff
@@ -26,7 +27,10 @@ export class HttpClient {
     private readonly clientId: string,
     private readonly clientSecret: string,
   ) {
-    this.axiosInstance = axios.create({ baseURL, timeout: 60000 });
+    this.axiosInstance = setupCache(
+      axios.create({ baseURL, timeout: 60000 }),
+      { ttl: 5 * 60 * 1000 }, // 5 min cache for get requests
+    );
```

Comment:
The only way to fix this that I see is to have a cache that does not only invalidate by time but also when there is a write to a certain asset/ts/whatever. So in the ts usecase the ts value would be chached at first but once the new value is written to the ts, the cache is invalidated. 
This will be really complicated and likely creates almost undebuggable problems.

Reply:
For fast charging data like time series, I am on your side. For the latest value interface I will disable the cache. But for everything else 5 min is not to long.
If you use the database as short memory for your flow, than you have an anti pattern implemented.
If you really need uncached data or a shorter cache time, than you can add a custom configuration to your API call (e.g. in the unit tests).